internal/factsengine/gatherers/packageversion_test.go

```diff
@@ -71,10 +71,16 @@ func (suite *PackageVersionTestSuite) TestPackageVersionGathererNoArgumentProvid
 }
 
 func (suite *PackageVersionTestSuite) TestPackageVersionGather() {
-	suite.mockExecutor.On("Exec", "rpm", "-q", "--qf", "%{VERSION}", "corosync").Return(
+	suite.mockExecutor.On("Exec", "/usr/bin/rpm", "-q", "--qf", "%{VERSION}", "corosync").Return(
```

Comment:
Can we add some new test to check that errors during the comparison command work as expected? In fact, the test code doesn't check any scenario for failed commands